src/main/scala/essential_scala/Ex4Traits.scala

```diff
@@ -194,13 +194,22 @@ object Ex4Traits extends App {
   t1.prettyPrint("")
   println(t1.sum())
   t1.double().prettyPrint("")
+
+  // 4.2.2.2 The Color and the Shape
+  val darkBlue = Colour(0, 0, 102)
+  val lightYellow = Colour(255, 255, 204)
+  println("darkBlue id dark: " + darkBlue.isDark)
```

Comment:
```suggestion
  println(s"darkBlue id dark: ${darkBlue.isDark}")
```

Reply:
OK, in this simple case that's easy to read.